openmrs/src/main/java/org/motechproject/openmrs/service/impl/OpenMRSObservationServiceImpl.java

```diff
@@ -117,7 +117,20 @@ public Observation createObservation(String configName, Observation observation)
             Observation created = obsResource.createObservation(config, observation);
             eventRelay.sendEventMessage(new MotechEvent(EventKeys.CREATED_NEW_OBSERVATION_SUBJECT, EventHelper.observationParameters(created)));
             return created;
+        } catch (HttpClientErrorException e) {
+            LOGGER.error("Error while creating observation!");
```

Comment:
exception handling should be better. I expect that this action will fails really often. Additionally we should throw exception because we want to task fail if this situation occurs